walker_lake/walker_lake.ipynb

```diff
@@ -131,15 +135,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(\"The shape of the DataArray is :\", landsat_5_img.shape)\n",
-    "print(\"With attributes:\\n \", '\\n  '.join('%s=%s'%(k,v) for k,v in landsat_5_img.attrs.items()))"
+    "assert landsat_5_img.rio.crs == landsat_8_img.rio.crs\n",
+    "print(landsat_5_img.rio.crs)"
```

Comment:
There should be a friendly message if you think the `crs` object is to be shown here. Looks like it might be a stray debug print to me right now...